sound/soc/intel/boards/sof_sdw.c

```diff
@@ -845,6 +860,7 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 	struct snd_soc_dai_link_component *codecs;
 	int cpu_dai_id[SDW_MAX_CPU_DAIS];
 	int cpu_dai_num, cpu_dai_index;
+	static int link_id = 0;
```

Comment:
> can we try to avoid using a static here?

I used static is because create_sdw_dailink() will be called multiple time and I want to keep the value of link_id. But I don't like static inside a function either. Using link_id as an argument of create_sdw_dailink() now.